src/ontology/edits/oeo-physical.omn

```diff
@@ -1620,15 +1620,20 @@ pull request:  https://github.com/OpenEnergyPlatform/ontology/pull/524
 
 alternative term electricity:
 issue: https://github.com/OpenEnergyPlatform/ontology/issues/381
-pull request: https://github.com/OpenEnergyPlatform/ontology/pull/621",
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/621
+
+add commodity role:
+issue: https://github.com/OpenEnergyPlatform/ontology/issues/891
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1022",
         rdfs:label "electrical energy",
         
             Annotations: <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/636
 pull request: https://github.com/OpenEnergyPlatform/ontology/pull/810"
         owl:equivalentClass "http://purl.obolibrary.org/obo/ENVO_2000020"
     
     SubClassOf: 
-        OEO_00000150
+        OEO_00000150,
+        <http://purl.obolibrary.org/obo/RO_0000087> some OEO_00040011
```

Comment:
> After implementing #891 / PR #1022 I saw your comment above @l-emele. We should revert the axiom `electrical energy has role some fuel role` and add the proposed one instead, when implementing this issue ❗

_Originally posted by @stap-m in https://github.com/OpenEnergyPlatform/ontology/issues/998#issuecomment-1034935419_

Is this axiom needed that this PR is consistent? Else I suggest to revert is already in this PR. So we cannot forget to change it later.

Reply:
I decided not to include this here but opened #1030 to discuss that a litte more profoundly.